Sources/VLCActionSheet.swift

```diff
@@ -0,0 +1,422 @@
+/*****************************************************************************
+ * VLCActionSheet.swift
+ *
+ * Copyright © 2018 VLC authors and VideoLAN
+ * Copyright © 2018 Videolabs
+ *
+ * Authors: Soomin Lee <bubu@mikan.io>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import Foundation
+import UIKit
+
+// MARK: VLCRendererCollectionViewLayout
+class VLCRendererCollectionViewLayout: UICollectionViewFlowLayout {
```

Comment:
This belongs in it's own class

Reply:
do you actually need a subclass here ? can't you just set that on a normal UICollectionViewFlowlayout ?

Reply:
the customization is too tiny to justify it's own class